packages/language-service/src/lib/css-service.ts

```diff
@@ -136,6 +136,7 @@ export class CssService {
                     diag.code === 'unknownAtRules' &&
                     (atRuleName === '@custom-selector' ||
                         atRuleName === '@st-scope' ||
+                        atRuleName === '@st-namespace' ||
```

Comment:
might need a change in `stylable.tmLanguage.json` stylable-intelligence

Reply:
I think `stylable.tmLanguage.json` is just for coloring and should fallback to regular at-rule, anyhow not part of this PR